pkg/scheduler/scheduler.go

```diff
@@ -309,9 +302,6 @@ func initPolicyFromConfigMap(client clientset.Interface, policyRef *schedulerapi
 
 // Run begins watching and scheduling. It waits for cache to be synced, then starts scheduling and blocked until the context is done.
 func (sched *Scheduler) Run(ctx context.Context) {
-	if !cache.WaitForCacheSync(ctx.Done(), sched.scheduledPodsHasSynced) {
```

Comment:
do we need to do something like `informerFactory.WaitForCacheSync(ctx.Done())`

Reply:
The caller already did it via informerFactory.